nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/provenance/ui/provenance-event-listing/provenance-event-table/provenance-event-table.component.html

```diff
@@ -17,7 +17,7 @@
 
 <div class="provenance-event-table h-full flex flex-col gap-y-2">
     <error-banner></error-banner>
-    <div [class.hidden]="showLineage" class="h-full flex flex-col gap-y-2">
+    <div [class.hidden]="showLineage" class="h-full flex flex-col">
```

Comment:
This change makes the filter controls too close to the table when compared to other tables with filter controls.

![Screenshot 2024-05-10 at 2 00 15 PM](https://github.com/apache/nifi/assets/123395/081ed13b-9206-43ac-a52a-bf3be9cc86b8)

Reply:
It also makes the refresh control at the bottom of the table too close.

Reply:
This is also an issue throughout the app. For all `mat-form-field`'s like this angular adds a bottom margin for the `mat-hint` or `mat-error`. However, in these usages of `mat-form-field` above tables throughout nifi we do not follow the material spec and instead we place the hint above the fields.

In order to address this issue I see two options:

1. We can update the UX to follow the material specification and leverage the `mat-hint`.

OR

2. We need to remove the margin at the bottom of these fields ( I think we can leverage https://material.angular.io/components/form-field/api#SubscriptSizing) and then also go through and update the spacing/padding of these form fields table filters.

Reply:
Ok, I have updated the layouts to display filter hint text below the form fields.

Reply:
My comment was regarding this change (not other parts of the application) specifically which makes these elements even closer. Usage of `hint` is a great idea. Let me review these changes in your latest commit.

Reply:
FYI I didn't actually use `<mat-hint>` but rather I simply moved the `<div>` to the appropriate place such that there are no overflow issues. This approach does not really correlate the 'hint' to a particular field which is why I didn't opt to use `<mat-hint>`. It actually doesn't make any visual difference whether we use `<mat-hint>` or `<div>` so if you like I can update to use `<mat-hint>`.